tools/distpartitioning/utils.py

```diff
@@ -63,6 +63,28 @@ def read_json(json_file):
 
     return val
 
+def get_etype_featnames(etype_name, schema_map):
+    """
+    Retrieves edge feature names for a given edge_type
+
+    Parameters:
+    -----------
+    eype_name : string
+        a string specifying a edge_type name
+
+    schema : dictionary
+        metadata json object as a dictionary, which is read from the input
+        metadata file from the input dataset
+
+    Returns:
+    --------
+    list : 
+        a list of feature names for a given edge_type
+    """
+	edge_data = schema_map[constants.STR_EDGE_DATA]
+	feats = edge_data.get(etype_name, {})
+	return [feat for feat in feats]
```

Comment:
Wrong indentation.

Reply:
indentation was correct. Except that the tabstop space was set incorrectly. It is an issue with the github UI interpreting the tapstop space. Now corrected it. hopefully it works.